Dependencies issue

Error: Cannot find module 'gulp-less'
`gulp-less` isn't included in your package.json, so gulp tasks won't work. Simple fix!
Edit: Add `gulp-plumber` to the list. `npm start` does work correctly after including those 2.

--- a/package.json
+++ b/package.json
@@ -42,6 +42,8 @@
     "watchify": "^2.1.1"
   },
   "dependencies": {
-    "react": "^0.12.0"
+    "react": "^0.12.0",
+    "gulp-plumber": "~0.6.6",
+    "gulp-less": "~3.0.1"
   }
 }

